### ISSUE TITLE
MultilineMethodCallIndentation inconsistently applied

### ISSUE THREAD

**Issue by jnardone** [rubocop-hq/rubocop#4431]
New with 0.49

Consider this code:

```
      Models::EventData.where(state: 'new')
        .exclude(actor_id: nil)
        .order(:created_at)
        .limit(50)
```

0.49 complains about:
```
C: Layout/MultilineMethodCallIndentation: Align .limit with .where on line 29.
        .limit(50)
```

But if I align all lines to the .where:

```
      Models::EventData.where(state: 'new')
                       .exclude(actor_id: nil)
                       .order(:created_at)
                       .limit(50)
```

```
C: Layout/MultilineMethodCallIndentation: Use 2 (not 17) spaces for indenting an expression spanning multiple lines.
                       .exclude(actor_id: nil)
C: Layout/MultilineMethodCallIndentation: Use 2 (not 17) spaces for indenting an expression spanning multiple lines.
                       .order(:created_at)
```

The only way I can appease the cop is to (a) indent the first two lines 2 spaces, and (b) indent the last line all the way to the .where, which is really unreadable.

**Comment by Drenmi** [comment]
What does your `.rubocop.yml` look like? When I run `Style/MultilineMethodCallIndentation` on your example on `master`, it auto-corrects to:

```ruby
Models::EventData.where(state: 'new')
                 .exclude(actor_id: nil)
                 .order(:created_at)
                 .limit(50)
```

and stops reporting violations there.

**Comment by jnardone** [comment]
```
AllCops:
  TargetRubyVersion: 2.4
  Exclude:
    - 'test/*'
    - 'vendor/**/*'

Metrics/CyclomaticComplexity:
  Max: 12

Metrics/PerceivedComplexity:
  Max: 12

Metrics/BlockLength:
  Exclude:
    - 'spec/**/*'
    - 'db/migrations/*'
    - 'api/**/*'

Metrics/AbcSize:
  Max: 30
  Exclude:
    - 'spec/**/*'

Metrics/ClassLength:
  Max: 125
  Exclude:
    - 'spec/**/*'

Metrics/LineLength:
  Max: 115

Metrics/ModuleLength:
  Exclude:
    - 'spec/**/*'

Metrics/MethodLength:
  Max: 20

Metrics/ParameterLists:
  CountKeywordArgs: false

Layout/LeadingCommentSpace:
  Exclude:
    - 'config.ru'

Style/MixinGrouping:
  Exclude:
    - 'spec/**/*'

```

**Comment by jnardone** [comment]
The MultilineMethodCallIndentation file that triggered was in "./lib" (re: exclusions above)

**Comment by jnardone** [comment]
Per #4476 it may be useful to note this method is a `private_class_method` declared method...

```
    private_class_method def self.retrieve_things
      Models::EventData.where(state: 'new')
        .exclude(actor_id: nil)
        .order(:created_at)
        .limit(50)
    end
```

**Comment by Drenmi** [comment]
@jnardone: That explains it. 🙂 Most of the indentation cops have not been adjusted to account for inline `def` modifiers yet.
